geode-core/src/test/java/org/apache/geode/internal/monitoring/ThreadsMonitoringProcessJUnitTest.java

```diff
@@ -34,11 +31,13 @@
  */
 public class ThreadsMonitoringProcessJUnitTest {
 
+  private static final int TIME_LIMIT = 1000;
```

Comment:
Please indicate the units of measure in the name or an additional parameter.

Reply:
done